src/Microsoft.Framework.CodeGeneration.Common/CommonUtilities.cs

```diff
@@ -6,14 +6,12 @@
 using System.Reflection;
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.Emit;
-using Microsoft.Framework.Runtime;
 
 namespace Microsoft.Framework.CodeGeneration
 {
     internal static class CommonUtilities
     {
         public static CompilationResult GetAssemblyFromCompilation(
-            IAssemblyLoadContext loader,
```

Comment:
@bricelam  and I were discussing this and he had used this approach for his work on T4 templating.